### PR TITLE
Updated Tags.json

### PR DESCRIPTION
**Pull request by ShinyaHir**
6969 420SussyBalls Amogus

### CODE REVIEW

**Comment by KTibow** [comment]
please use proper formatting and refrain from making "jokes"

**Comment by ShinyaHir** [comment]
fixed confilct

**Comment by ShinyaHir** [comment]
changed name so its not a joke anymore

**Comment by KTibow** [comment]
that wasn't what i was referring to

**Comment by ShinyaHir** [comment]
what are you talking about then?

File: docs/assets/tags.json [hunk 1]
```diff
@@ -29,7 +29,8 @@
     "kti": ["&9&Z[skyclient dev]&r", "&9&Z[SC dev]&r"],
     "alfredo": ["&7[Alfredo cuz my friend is eating it]&r", "&7[Alfredo]&r"],
     "Maya": ["&8[&dCutie&8]", "&8[&5Maya&8]"],
-    "Zetvue": ["&8[&c♫i make music♫&8]&r", "&8[&cZetvue&8]&r"]
+    "Zetvue": ["&8[&c♫i make music♫&8]&r", "&8[&cZetvue&8]&r"],
+    "Shinya": ["&5&l[&d&lMega Omega Weeb 6969 420SussyBalls Amogus&5&l]","&5&l[&d&lWeeb&5&l]"]
```
Comment (KTibow) [inline docs/assets/tags.json:33]: do you find this "humorous"

**Review by KTibow — CHANGES_REQUESTED** [review]

File: docs/assets/tags.json (hunk 1 above)
Comment (Zetvue) [reply docs/assets/tags.json:33]: shinya on his way to make the most unfunny jokes anyone has ever heard

Comment (ShinyaHir) [reply docs/assets/tags.json:33]: uh, yes?
its fune

**Comment by KTibow** [comment]
stale

**Pull request closed without merging**